### PR TITLE
Changes for 2.3.0 beta & documentation

### PR DESCRIPTION
**Pull request by tahmmee**

### CODE REVIEW

File: tools/value-generation/gen.py [hunk 1]
```diff
@@ -134,6 +145,9 @@ def processCluster(crd_value, value_map, comment_map) :
   # Set this empty to ensure we auto-generate it by default
   value_map[crd_value]['security']['adminSecret'] = ''
 
+  # Set monitoring to emtpy since it requires image which is not provided by default
+  value_map[crd_value]['monitoring'] = {}
```
Comment (patrick-stephens) [inline tools/value-generation/gen.py:149]: We should probably sort that soon

File: tools/value-generation/README.md [hunk 2]
```diff
@@ -3,11 +3,10 @@
 This directory contains simple Python tooling to auto-generate a `values.yaml` file with comments from a CRD.
 
 `docker build -t values-generator .`
-`docker run values-generator > values.yaml`
 
 The YAML is output to standard output of the container.
 To run with a custom CRD:
-`docker run values-generator - < <CRD>`
+`docker run values-generator -f full - < <CRD>`
```
Comment (patrick-stephens) [inline tools/value-generation/README.md:9]: I can't see any changes to handle this flag - did we miss it before?

Comment (tahmmee) [reply tools/value-generation/README.md:9]: Right it's set to "full" by default, so having it explicit could document that it's being used

**Review by patrick-stephens — APPROVED** [review]

File: tools/value-generation/gen.py (hunk 1 above)
Comment (tahmmee) [reply tools/value-generation/gen.py:149]: Thinking the operator should provide default image?

Comment (patrick-stephens) [reply tools/value-generation/gen.py:149]: Not part of this task though.

Comment (tahmmee) [reply tools/value-generation/gen.py:149]: Agree

**Pull request merged**